packages/mermaid/src/diagram-api/types.ts

```diff
@@ -79,8 +79,10 @@ export type DrawDefinition = (
  */
 export type ParseDirectiveDefinition = (statement: string, context: string, type: string) => void;
 
-export type HTML = d3.Selection<HTMLIFrameElement, unknown, Element, unknown>;
+export type HTML = d3.Selection<HTMLIFrameElement, unknown, Element | null, unknown>;
 
-export type SVG = d3.Selection<SVGSVGElement, unknown, Element, unknown>;
+export type SVG = d3.Selection<SVGSVGElement, unknown, Element | null, unknown>;
+
+export type Group = d3.Selection<SVGGElement, unknown, Element | null, unknown>;
```

Comment:
It may be insignificant, but I would rename it Group, because Group is ubiquitous and too abstract. If it is an html element that contains that SVG, it could be SVGParent, if it is <g> inside svg it could be SVGGroup, or even G (yep, one-letter type name). I am afraid Group is too broad

Not necessary to change this, simply asking

Reply:
It's should be assignable to return type of `svg.append('g')`, which is the `g` tag inside an `svg`.

I wasn't really sure what to call it. Originally, it was `G`, but I wasn't really sure does that `G` represents until I [searched](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/g) and found it's suppose to be `group`.

Reply:
Ok, lets try this